extensions/quack/description.yml

```
@@ -10,4 +10,10 @@ extension:
 
 repo:
   github: hannes/quack
-  ref: 09680a975bcf9e93b5a2f46d3eeb68792d5239c6
\ No newline at end of file
+  ref: 09680a975bcf9e93b5a2f46d3eeb68792d5239c6
+
+docs:
+  hello_world: |
+    SELECT quack(i::VARCHAR) FROM range(10) tbl(i);
```

nit: maybe this can actually quack hello world at the user?
```suggestion
    SELECT quack('world');
```

this outputs :
```
┌────────────────┐
│ quack('world') │
│    varchar     │
├────────────────┤
│ Quack world 🐥 │
└────────────────┘
```
which seems cute for a hello world example